### PR TITLE
Merge pull request  for CCD-436 Improve CCD Performance

### PR DESCRIPTION
**Pull request by OlanrewajuAdedeji**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CCD-436

### Change description ###
remove duplicate indexes from def store db

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: repository/src/main/resources/db/migration/V20210716_1436__RDM-1150_Drop_duplicate_indexes.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+DROP INDEX idx_reference  CASCADE;
```
Comment (danlysiak) [inline repository/src/main/resources/db/migration/V20210716_1436__RDM-1150_Drop_duplicate_indexes.sql:1]: Could we please use `IF EXISTS` and without `CASCADE` for all of these e.g.:
```suggestion
DROP INDEX IF EXISTS idx_reference;
```

File: repository/src/main/resources/db/migration/V20210716_1436__RDM-1150_Drop_duplicate_indexes.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+DROP INDEX idx_event__case_type_id  CASCADE;
```
Comment (danlysiak) [inline repository/src/main/resources/db/migration/V20210716_1436__RDM-1150_Drop_duplicate_indexes.sql:2]: Please double-check the name of this file; it doesn't seem to match the convention:
`V20210716_1436__RDM-1150_Drop_duplicate_indexes.sql` should be
`V20210716_436__CCD-436_Drop_duplicate_indexes.sql`

File: repository/src/main/resources/db/migration/V20210716_1436__RDM-1150_Drop_duplicate_indexes.sql [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+DROP INDEX event_pre_state_state_id_event_id_idx  CASCADE;
+DROP INDEX idx_field_type__id__base_field_type_id__collection_field_type_i  CASCADE;
+DROP INDEX idx_jurisdiction__id__reference  CASCADE;
+DROP INDEX idx_field_type_list_item__field_type_id  CASCADE;
```
Comment (danlysiak) [inline repository/src/main/resources/db/migration/V20210716_1436__RDM-1150_Drop_duplicate_indexes.sql:6]: I've run the same SQL to identify duplicate indexes locally and see 8, but there's only 6 in here. Were the others intentionally excluded?

File: repository/src/main/resources/db/migration/V20210716_1436__RDM-1150_Drop_duplicate_indexes.sql (hunk 1 above)
Comment (OlanrewajuAdedeji) [reply repository/src/main/resources/db/migration/V20210716_1436__RDM-1150_Drop_duplicate_indexes.sql]: SQL Statements update, by removing CASCADE and adding IF EXISTS

File: repository/src/main/resources/db/migration/V20210716_1436__RDM-1150_Drop_duplicate_indexes.sql (hunk 2 above)
Comment (OlanrewajuAdedeji) [reply repository/src/main/resources/db/migration/V20210716_1436__RDM-1150_Drop_duplicate_indexes.sql]: Update/refactor File name to match convention

**Review by danlysiak — APPROVED** [review]
👍

**Pull request merged**